releasenotes/notes/dogstatsd-replay-nommap-9648eef0f6f47488.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Allow dogstatsd replay files to be fully loaded into memory as opposed
+    to relying on MMAP. We still default to MMAPing replay targets.
```

Comment:
Nit: 
```suggestion
    to relying on mmap(2). We still default to using mmap(2) for replay targets.
```

Reply:
The comment doesn't quite paint the multi-platform picture. Maybe this should state "memory map" so that it's more accurate (unfortunately `mmap(2)` doesn't quite work for windows)?

Reply:
Hm, good point.  "Memory map" is fine, or leaving it as-is is fine too.  The "changes requested" is for the unmapping issue, not this one :)